lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,117 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
+
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+  const renderEl = rtlRenderElement(NxPageHeader, {});
+
+ it('renders a top-level header element', function() {
+    renderEl();
+    const header = screen.getByRole('banner');
+    expect(header).toBeInTheDocument();
```

Comment:
Since the test specifically refers to the "top-level" element
```suggestion
    const el = renderEl();
    const header = screen.getByRole('banner');
    expect(header).toBe(el);
```